app/src/androidTest/java/org/mozilla/fenix/ui/CustomTabsTest.kt

```diff
@@ -27,7 +27,7 @@ import org.mozilla.fenix.ui.robots.searchScreen
 class CustomTabsTest {
     private lateinit var mockWebServer: MockWebServer
     private val customMenuItem = "TestMenuItem"
-    private val externalLinksPWAPage = "https://mozilla-mobile.github.io/testapp/externalLinks.html"
+    private val externalLinksPWAPage = "https://andiaj.github.io/testapp/v2.0/externalLinks.html"
```

Comment:
I think you forgot to change back to the mozilla-mobile domain

Reply:
Sure, first I want to get approval for the versioning update.